capture/run_tasks_pypln.py

```diff
@@ -0,0 +1,28 @@
+from celery import Celery
+import pypln.api
+import pymongo
+import settings
+from tasks_pypln import fetch_property
+from celery import group
+
+
+## Media Cloud database setup
+client = pymongo.MongoClient(host=settings.MONGOHOST)
+articles = client.MCDB.articles # articles collection
+pypln_temp = client.MCDB.pypln_temp # pypln_temp temporary collection
+articles_analysis = client.MCDB.articles_analysis # articles_analysis collection
+
+
+def get_pypln_properties(doc_id):
+    article = pypln_temp.find_one({'_id': doc_id})
+    articles_analysis.insert({'articles_id': article['articles_id']})
+
+    results = fetch_property.delay(doc_id)
+
+    pypln_temp.remove({'_id': doc_id})
```

Comment:
I thought about that, but than I realized that celery has a backend that stores all tasks status. So if something happen while running the tasks, celery it self will reload the task, that way we'll not loose the document that was removed.